host/xtest/pkcs11_1000.c

```diff
@@ -2565,3 +2565,237 @@ static void xtest_pkcs11_test_1011(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(pkcs11, 1011, xtest_pkcs11_test_1011,
 		 "PKCS11: Test Find Objects");
+
+
+static void xtest_pkcs11_test_1012(ADBG_Case_t *c)
+{
+	CK_RV rv = CKR_GENERAL_ERROR;
+	CK_SLOT_ID slot = 0;
+	CK_SESSION_HANDLE session = CK_INVALID_HANDLE;
```

Comment:
I suggest to rename `session` to `rw_session` and `session_flags` to `rw_session_flags` for consistency with `ro_session` and `ro_session_flags` variable labels.

Reply:
Renamed